README.md

```diff
@@ -56,56 +56,6 @@ RUNTIME_MODE       | no       | production               | Running mode (develop
 - Environment variables can also be set in the `./config.json` for development. If both provide a value for a specific property, the environment variable overrides the file. You can define your own `config.json` file and pass it to the application with the following command: `-c <config_file>`
 - The application can take any flags for [glog](https://github.com/golang/glog), which passes them straight into glog. The glog flag `--logtostderr` is set to true by default.
 
-### Dev Preview (Search Configurable Collection)
```

Comment:
Where is this topic moving to?
To avoid breaking existing links, let's add a pointer to the new document from here. We also need to keep the title unchanged.

```
### Dev Preview (Search Configurable Collection)
Configurable collection is now fully supported. This topic has moved to <add link here>.
```

Reply:
I was thinking we can add to wiki within search-collector for now, unless there is a better place for it. okay, will keep the title and link to wiki until the product doc is ready.

Reply:
Yes, the search-v2-operator wiki works.